source_sdk/entity/entity.h

```diff
@@ -36,3 +39,19 @@ enum
 	TF_CLASS_SPY,
 	TF_CLASS_ENGINEER
 };
+
+typedef struct player_info_s
+{
+	char			name[32];
+	int			userID;
+	char			guid[33];
+	unsigned long	        friendsID;
+	char			friendsName[32];
+	bool			fakeplayer;
+	bool			ishltv;
+	unsigned long	customFiles[4];
+	unsigned char	filesDownloaded;
+} player_info_t;
```

Comment:
I haven't been using `typedef struct` for the other structs so I'd remove this here and use the fully qualified name. Also I haven't been space aligning structs so I'd remove these spaces.

Reply:
Also, not important, but I don't believe we don't need the other members of the struct if we aren't using them. Especially since the member we want is the first member. Worth keeping though since these might be useful later.

Reply:
Removed typedef

Reply:
> I haven't been using `typedef struct` for the other structs so I'd remove this here and use the fully qualified name. Also I haven't been space aligning structs so I'd remove these spaces.

That makes sense. I had copied it from the official SDK and never thought about it like that. I'll keep it in mind.